AstroSaveConverter.py

```diff
@@ -1,182 +1,187 @@
-                f'Please use only values between 1 and {max_save_number} or 0 alone')
-    return save_numbers_list
-
-
-def manage_rename(container):
-    """
-    Manage wether some saves have to be renamed or not
-
-    Arguments:
-        container -- Container from which to rename the saves
-
-    Returns:
-        Nothing 
-
-    Exception:
-        None
-    """
-    is_rename = None
-    while is_rename != 'y' and is_rename != 'n':
-        AstroLogging.logPrint('\nWould you like to rename a save ? (y/n)')
-        is_rename = input().lower()
-
-    if is_rename == 'y':
-        for number in save_numbers_list:
-            container.save_list[number - 1].rename()
-
-
-if __name__ == "__main__":
-    try:
-        os.system(
-            "title AstroSaveConverter - Migrate your Astroneer save from Microsoft to Steam")
-    except:
-        pass
-    try:
-        parser = argparse.ArgumentParser()
-        parser.add_argument(
-            "-f", "--fileContainer", help="File container to parse", required=False)
-
-        args = parser.parse_args()
-
-        path = os.getcwd()
-        # TODO verifier si le dossier courant est un sous-dossier de %appadata% et warning
-        # TODO quelquepart: si qqun rentre "connard" dans un prompt, insulter copieusement EmptyProfile
-        AstroLogging.setup_logging(path)
-
-        try:
-            fileContainer = check_container_path(args.fileContainer)
-        except FileNotFoundError as e:
-            AstroLogging.logPrint('\nNo container found, press a key to exit')
-            input()
-            sys.exit(-1)
-
-        container = AstroSaveContainer(fileContainer)
-
-        AstroLogging.logPrint('Container file loaded successfully !\n')
-        container.print_container()
-
-        save_numbers_list = choose_save_to_export(container)
-
-        manage_rename(container)
-
-        AstroLogging.logPrint(f'\nExtracting saves {str(save_numbers_list)}')
-
-        container.xbox_to_steam(save_numbers_list)
-
-        AstroLogging.logPrint(f'\nTask completed, press any key to exit')
-        input()
-    except Exception as ex:
-        AstroLogging.logPrint(ex)
-        AstroLogging.logPrint('', 'exception')
-
-        sys.exit()
+import argparse
+import os
+import sys
+
+from cogs.AstroLogging import AstroLogging
+from cogs.AstroSaveContainer import AstroSaveContainer
+"""
+D
+
+I
+
+Arguments:
+
+Returns:
+    Returns 
+
+Exception:
+    None
+"""
+
+
+def check_container_path(path):
+    """
+    Defines the container to use
+
+    If the path variable is empty, find every *container* file in the
+    current directory and make the user chose one
+
+    Arguments:
+        path -- current known path for the container
+
+    Returns:
+        Returns "path" if it was non-empty
+        Else returns the chosen container
+
+    Exception:
+        Raise FileNotFoundError if no container is found
+    """
+    list_containers = []
+    
+    if path==' ' or path==None:

```

Comment:
Why check for blank space and not empty string ?

Reply:
sorry it's was path==''